INC CREATEv1

```diff
@@ -0,0 +1,197 @@
+        password.send_keys("Liverpool28")
+        password.send_keys(Keys.RETURN)
+
+        identifier_inc = ("AUTOMATED TESTING - INCIDENT - %s" %today)
+
+        # depreciated warning
+        driver.switch_to.default_content()
+        # driver.switch_to.frame
+
+        time.sleep (10)
+
+        # Left Nav, incident.do 
+        createnew = driver.find_element_by_id("filter")
+        createnew.send_keys("Create New")
+        time.sleep(1)
+        createnew.send_keys(Keys.RETURN)
+        time.sleep(1)
+
+        # switch to main frame to create the incident
+        driver.switch_to_frame("gsft_main")
+
+
+        try:
+            element = WebDriverWait(driver, 10).until(
+            EC.presence_of_element_located((By.ID, "incident.category"))
+                
+                )
+        except:
+            print("Element not found")
+                
+            #driver.quit()
+
+        category = driver.find_element_by_id("incident.category")
+        category.send_keys("Software")
+        time.sleep(1)
+
+
+        subcategory = driver.find_element_by_id("incident.subcategory")
+        subcategory.send_keys("Email")
+        time.sleep(1)
+
+        time.sleep(2)
+        callerbox = driver.find_element_by_id("sys_display.incident.caller_id")
+        callerbox.send_keys("Abraham Lincoln")
+        callerbox.send_keys(Keys.RETURN)
+        time.sleep(1)
+
+        contacttype = driver.find_element_by_id("incident.contact_type")
+        contacttype.send_keys("Email")
+        time.sleep(1)
+
+        impact = driver.find_element_by_id("incident.impact")
+        impact.send_keys("2")
+        time.sleep(1)
+
+        urgency = driver.find_element_by_id("incident.urgency")
+        urgency.send_keys("2")
+        time.sleep(1)
+
+        assign_grp = driver.find_element_by_id("sys_display.incident.assignment_group")
+        assign_grp.send_keys("Database")
+        assign_grp.send_keys(Keys.RETURN)
+
+        time.sleep(1)
+
+
+        short_desc = driver.find_element_by_id("incident.short_description")
+        short_desc.send_keys("%s %s %s %s" %(identifier, module, action, today))
+        time.sleep(1)
+
+        desc = driver.find_element_by_id("incident.description")
+        desc.send_keys("Here are some notes")
+        time.sleep(1)
+
+
+        # Click the save button, brings you back to the list view
+        driver.find_element_by_id("sysverb_insert").click()
+
+        # In the List View, Now commence search, Click on the funnel
+        funnel = driver.find_element(By.XPATH,'//*[@id="incident_filter_toggle_image"]')
+        funnel.click()
+        time.sleep(2)
+
+        # List View - Next Step
+        select1 = driver.find_element(By.XPATH, '//*[@id="select2-chosen-4"]')
+        select1.click()
+        time.sleep(2)
+
+        # List View - Type the field to search on
+        search_box = driver.find_element(By. XPATH, '//*[@id="s2id_autogen4_search"]')
```

Comment:
keep variable name consistent:
e.g. either search_box or searchbox
best approach would be underscore separated
see PEP8 standards:
https://www.python.org/dev/peps/pep-0008/